### PR TITLE
Update NFS-Ganesha GlusterFS Integration.md

### PR DESCRIPTION
**Pull request by thotz**

### CODE REVIEW

File: Administrator Guide/NFS-Ganesha GlusterFS Integration.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ Note :
 Step 4 can be performed via following script
 #/usr/libexec/ganesha/dbus-send.sh <ganesha directory> [on|off] <volume name>
 ```
+Above scripts(meniotned in step 3 and step 4) is only available in glusterfs 3.10 rpms
```
Comment (prashanthpai) [inline Administrator Guide/NFS-Ganesha GlusterFS Integration.md]: typo - `mentioned` and `are available`

File: Administrator Guide/NFS-Ganesha GlusterFS Integration.md [hunk 2]
```diff
@@ -119,6 +121,9 @@ Step 4 can be performed via following script
        - *#dbus-send --type=method_call --print-reply --system --dest=org.ganesha.nfsd /org/ganesha/nfsd/ClientMgr org.ganesha.nfsd.clientmgr.ShowClients*
 
 ## Using Highly Available Active-Active NFS-Ganesha And GlusterFS cli
+
+### Please Note currently HA solution for nfs-ganesha is only available in 3.10 From 3.12 onwards HA will be handled by different project known as [storhaug](https://github.com/linux-ha-storage/storhaug) which is under development.
```
Comment (prashanthpai) [inline Administrator Guide/NFS-Ganesha GlusterFS Integration.md]: Please add punctuation to this sentence and do not use a markdown heading. You can use a [markdown Blockquotes](https://daringfireball.net/projects/markdown/syntax#blockquote) for this.

File: Administrator Guide/NFS-Ganesha GlusterFS Integration.md (hunk 1 above)
Comment (thotz) [reply Administrator Guide/NFS-Ganesha GlusterFS Integration.md]: done

File: Administrator Guide/NFS-Ganesha GlusterFS Integration.md (hunk 2 above)
Comment (thotz) [reply Administrator Guide/NFS-Ganesha GlusterFS Integration.md]: Done

**Pull request merged**